internal/services/compute/image_resource.go

```diff
@@ -54,6 +54,7 @@ func resourceImage() *pluginsdk.Resource {
 				ForceNew: true,
 			},
 
+			// TODO: rename to hyper_v_generation_version in v3.0
```

Comment:
(as below) we can remove this, the name is fine:

```suggestion
```

Reply:
Removed